From f32e5c03d18977d32b1eb5df2deb2a908ff4780c Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Mon, 24 Jun 2024 18:15:46 +0200
Subject: [PATCH] Fix build with newer ICU

Newer ICU requires C++17 but the checked-in generated parser code is
incompatible with that...

Needed to unbreak the Qt 5 CI seed jobs.
---
 CMakeLists.txt                      |  1 +
 src/parser/generated/sqlscanner.cpp | 40 ++++++++++++++---------------
 2 files changed, 21 insertions(+), 20 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 83801cce..634ab9e3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
 
 # Dependencies
+set(CMAKE_CXX_STANDARD 17)
 set(REQUIRED_QT_VERSION 5.4.0)
 set(REQUIRED_KF5_VERSION 5.45.0)
 
